modules/linear_boltzmann_solvers/executors/pi_keigen.cc

```diff
@@ -15,42 +15,37 @@ namespace opensn
 namespace lbs
 {
 
-OpenSnRegisterObjectInNamespace(lbs, XXPowerIterationKEigen);
+OpenSnRegisterObjectInNamespace(lbs, PowerIterationKEigen);
 
 InputParameters
-XXPowerIterationKEigen::GetInputParameters()
+PowerIterationKEigen::GetInputParameters()
 {
   InputParameters params = opensn::Solver::GetInputParameters();
 
-  params.SetGeneralDescription("Generalized implementation of a k-Eigenvalue solver using Power "
-                               "Iteration.");
+  /* clang-format off */
+  params.SetGeneralDescription("Implementation of a k-Eigenvalue solver using Power Iteration");
   params.SetDocGroup("LBSExecutors");
-
-  params.ChangeExistingParamToOptional("name", "XXPowerIterationKEigen");
-
+  params.ChangeExistingParamToOptional("name", "PowerIterationKEigen");
   params.AddRequiredParameter<size_t>("lbs_solver_handle", "Handle to an existing lbs solver");
   params.AddOptionalParameter("max_iters", 1000, "Maximum power iterations allowed");
   params.AddOptionalParameter("k_tol", 1.0e-10, "Tolerance on the k-eigenvalue");
-  params.AddOptionalParameter(
-    "reset_solution",
-    true,
-    "Flag, if set to true will initialize the phi-solution to all 1's before "
-    "executing");
-
-  params.AddOptionalParameter(
-    "reinit_phi_1", true, "If true, reinitializes scalar phi fluxes to 1");
+  params.AddOptionalParameter("reset_solution",
+                              true,
+                              "If set to true will initialize the flux moments to 1.0");
```

Comment:
```suggestion
  params.AddOptionalParameter(
    "reset_solution", true, "If true, reinitializes the flux moments to 1.0");
```

Reply:
So... this was the formatting I was trying to avoid. 😆  I prefer having everything formatted consistently, but if you'd prefer having everything on one line if it fits, I'm ok with that.

Reply:
If you prefer that, I can just turn clang-format back on, and it'll take care of everything.  Easy enough.

Reply:
If I'm not mistaken, our clang-format settings will put everything on the next line if it is less than 100 characters, otherwise it does one argument per line. I am good with whatever clang-format gives.